### PR TITLE
ARM templates for creating App Service Certificates

### PR DESCRIPTION
**Pull request by akurmi**
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Thank you for deciding to contribute to this repository, we're thrilled to have you contribute a template or a fix! :tada: Before we proceed, please be sure to review the contribution guidelines that will make the review process go smoother.
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

### Changelog
*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@akurmi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (akurmi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: app-service-certificate-standard/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+By default, 'Microsoft.CertificateRegistration' and 'Microsoft.Web' RPs don't have access to the Key Vault specified in the template hence you need to authorize these RPs by executing
```
Comment (singhkay) [inline app-service-certificate-standard/README.md]: Readme needs deploy to azure and visualize buttons. See one of the other templates

Comment (singhkay) [inline app-service-certificate-standard/README.md]: This template seems like it should be called 101-app-service-certificate-standard

File: app-service-certificate-standard/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
```
Comment (singhkay) [inline app-service-certificate-standard/azuredeploy.json]: for parmeters, prefix the prereqs with "existing" keyword

File: app-service-certificate-standard/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+         "allowedValues":[
+            "StandardDomainValidatedSsl",
+            "StandardDomainValidatedWildCardSsl"
+         ],
+         "defaultValue":"StandardDomainValidatedSsl",
+         "metadata":{
+            "description":"App Service Certificate type"
+         }
+      },
+      "keyVaultId":{
```
Comment (singhkay) [inline app-service-certificate-standard/azuredeploy.json]: you can probably ask the user for key vault resource group and name and build the resource id yourself using the resourceId function

Comment (akurmi) [inline app-service-certificate-standard/azuredeploy.json]: This is not possible as in order to create the required auth policies, user needs to call a graph API.

File: app-service-certificate-standard/README.md (hunk 1 above)
Comment (akurmi) [inline app-service-certificate-standard/README.md]: Fixed

File: app-service-certificate-standard/azuredeploy.json (hunk 2 above)
Comment (akurmi) [inline app-service-certificate-standard/azuredeploy.json]: Using this naming convention for App Service and Key Vault params.

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**